test/pycheck/motherduck_test.py

```diff
@@ -56,6 +56,37 @@ def test_md_create_table(md_cur: Cursor, ddb):
         md_cur.sql("CREATE TABLE t4(a int) USING heap")
 
 
+def test_md_read_scaling(pg: Cursor, ddb, default_db_name, md_test_user):
+    weird_db_name = "some 19 really $  - @ weird name 😀 84"
+    try:
+        ddb.sql(f'DROP DATABASE "{weird_db_name}";')
```

Comment:
```suggestion
        ddb.sql(f'DROP DATABASE "{weird_db_name}" IF EXISTS;')
```

Reply:
We very sadly don't support `IF EXISTS`

Reply:
It seems I put it in the wrong place:
```suggestion
        ddb.sql(f'DROP DATABASE IF EXISTS "{weird_db_name}";')
```

We use it in the `create_duckdb` function. So it at least seems like it works.

Reply:
Oh wow! I've always assumed we didn't... TIL!